test/relocatedepot.jl

```diff
@@ -0,0 +1,23 @@
+using Test
+
+
+include("testenv.jl")
+
+
+@testset "compile and load relocated pkg" begin
+    pkg = Base.identify_package("DelimitedFiles")
```

Comment:
`DelimitedFiles` should be in the stdlib path? So it should already be precompiled.

Reply:
True.

Should I instead add an external package to test the compilation step?
Maybe something lightweight like `UnPack.jl` or `Revise.jl`?

Reply:
You could also create your own external package (which we could move to the JuliaLang GitHub org) and use that in the tests. Then you could have maximum control over what exactly gets tested.

Reply:
You don't even need to make it external. Just create a small package here. We do it for other tests as well.

Reply:
Added a local Pkg and fixed the stdlib test.

Reply:
I am unsure about how useful the stdlib now test is, because IIUC then if the stdlib cannot be located at startup it will be recompiled. And by the time the test is run it is already cached again ...